### DISCUSSION TITLE
Lab6 firmware 合併問題

### DISCUSSION THREAD

**Discussion by yuchengliu0519** [bol-edu/HLS-SOC-Discussions#164] [Q&A]
大家好，我們現在想要把fir.c跟uart.c一起編譯成一個hex檔案，然後這個hex檔案有呼叫checkbit寫成AB40，但我們在波型圖裡面發現checkbit沒有變成AB40，甚至值完全沒變過。在我們單獨跑fir.c以及uart.c的時候功能都是正常的，想請教一下各位這會是哪些問題?
![image](https://github.com/bol-edu/HLS-SOC-Discussions/assets/47782190/cb8036dd-e8f7-42e8-b989-61257cc642dd)
下圖是正常跑的時候
![image](https://github.com/bol-edu/HLS-SOC-Discussions/assets/47782190/3d5ec55a-5dad-4db3-bd87-15f04e40db20)
下圖是兩個C code一起編譯後checkbit的結果
![image](https://github.com/bol-edu/HLS-SOC-Discussions/assets/47782190/ebf2157a-1f9e-415e-bb00-4ff521339af8)

**Comment by Josephood7** [comment]
去Include.rtl.list裡面
我把uart wrapper comment掉
把project wrapper解comment

**Reply by yuchengliu0519** [reply]
已成功，謝謝
